src/libraries/Microsoft.Extensions.HostFactoryResolver/pkg/Microsoft.Extensions.HostFactoryResolver.Sources.pkgproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <PropertyGroup>
```

Comment:
NIT: do we want this package to have it's own package version to decouple it from the rest of the product? SO maybe have it be like 1.0.0 or something?

Reply:
Another version means we need to maintain it.  It's fine to version with the project.